acknowledging.html

```diff
@@ -237,6 +237,18 @@ <h2 id="publication"><a href="#pub">In publications</a></h2>
             </li>
 
         </ul>
+	<p>
+	  We also provide the following LaTeX snippet to add to your acknowledgments: 
+	</p>
+	<div class="box">
+	  <code>
+  	    This work made use of Gammapy \citep{gammapy:2023}, a community-developed core Python package. The Gammapy team acknowledges all Gammapy past and 
+	    current contributors, as well as all contributors of the main Gammapy dependency libraries: <a href="https://numpy.org/">NumPy</a>,
+	    <a href="https://scipy.org/">SciPy</a>, <a href="http://www.astropy.org">Astropy</a>, 
```

Comment:
Do we want to have an acknowledgement within an acknowledgement? eg: the astropy acknowledgement does not include references to numpy etc (https://www.astropy.org/acknowledging.html)

Reply:
Indeed... We discussed that during some calls. In contrary to Astropy that is a "low-level" library, we are an "end-user" library that encompasses main others and we profit of their work. So, we thought that it is fair to cite them and important in order to respect a bit their acknowledgement policy.
Of course, thoughts are welcome...